src/app/page.tsx

```diff
@@ -1,101 +1,49 @@
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="/file.svg"
-            alt="File icon"
-            width={16}
-            height={16}
-          />
-          Learn
-        </a>
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="/window.svg"
-            alt="Window icon"
-            width={16}
-            height={16}
-          />
-          Examples
-        </a>
-        <a
-          className="flex items-center gap-2 hover:underline hover:underline-offset-4"
-          href="https://nextjs.org?utm_source=create-next-app&utm_medium=appdir-template-tw&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <Image
-            aria-hidden
-            src="/globe.svg"
-            alt="Globe icon"
-            width={16}
-            height={16}
-          />
-          Go to nextjs.org →
-        </a>
-      </footer>
-    </div>
-  );
+    return (
+        <>
+            <Input placeholder="Search" onChange={(e) => setFilter(e.target.value)}/>
+            <Table hideHeader>
+                <TableHeader>
+                    <TableColumn>Avatar</TableColumn>
+                    <TableColumn>Repository</TableColumn>
+                    <TableColumn>Link to repo</TableColumn>
+                    <TableColumn>Link to contributors</TableColumn>
+                </TableHeader>
+                <TableBody>
+                    {data?.map((item: GitHubRepo) => item.full_name.includes(filter) && (
+                        <TableRow key={item.id}>
+                            <TableCell>
+                                <Link href={item.owner.html_url}>
+                                    <Avatar src={item.owner.avatar_url} radius="none"/>
+                                </Link>
+                            </TableCell>
+                            <TableCell>{item.full_name}</TableCell>
+                            <TableCell>
+                                <Link href={item.html_url}>
+                                    Open
+                                </Link>
+                            </TableCell>
+                            <TableCell>
+                                <Link href={item.full_name}>
+                                    Browse Contributors
```

Comment:
```suggestion
                                    Show Core Contributors
```
In Github terminology, "Contributors" are anyone who has a commit in a repo. So, this might create some confusion. Making it explicit can provide clarity.

Reply:
That is exactly why I say Contributors and not Core Contributors, because we are not filtering and some of the people listed here will be admins etc who have access but are not CCs.

Reply:
Could we enable filtering by cross-referencing the global list of CCs from https://github.com/openedx/wg-coordination/issues/149? (not necessarily as part of this very first POC but in general)